capstone-app/src/app/modules/maps/map/map.component.spec.ts

```diff
@@ -17,4 +21,11 @@ describe('MapComponent', () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
   });
+
+
+  it('should create the app', () => {
+    // const fixture = TestBed.createComponent(MapComponent);
+    // const app = fixture.componentInstance;
+    expect(fixture.componentInstance).toBeTruthy();
+  });
 });
```

Comment:
Add tests that the map center is set correctly, and that the correct number of markers are displayed (along with their positions being correct).